### ISSUE TITLE
Figure attributes in fatiando.vis.mpl

### ISSUE THREAD

**Issue by Johumel** [fatiando/fatiando#426]
Hi,
I am using fatiando for the first time and I tried to run one of the cookbook examples (GravMag: 2D forward modeling with polygons) but wasn't successful due to this error:

AttributeError                            Traceback (most recent call last)
<ipython-input-1-e5945cb9735c> in <module>
      7 # This way, the z axis in the plots points down.
      8 area = (-5000, 5000, 5000, 0)
----> 9 axes = mpl.figure().gca()
     10 mpl.xlabel("X")
     11 mpl.ylabel("Z")

AttributeError: module 'fatiando.vis.mpl' has no attribute 'figure'

**Comment by leouieda** [comment]
Hi @Johumel thank you for posting! We have actually transitioned away from the `fatiando` package into having several different more specialized packages. See this blog post for context: https://www.leouieda.com/blog/future-of-fatiando.html

The error you have is because our old documentation wasn't updated when we removed the `mpl` module. This is one of the reasons why we decided to move away from the single `fatiando` package into much much better designed new packages. Have a look at what we to offer: https://www.fatiando.org/#projects
